src/main/java/com/iridium/iridiumskyblock/utils/PlayerUtils.java

```diff
@@ -95,10 +95,12 @@ public static void teleportSpawn(Player player) {
         if (Bukkit.getPluginManager().isPluginEnabled("EssentialsSpawn")) {
```

Comment:
Dont we already check for that here?

Reply:
API says the answer can be null, better avoid a problem in the future

Reply:
Crash Plugin, Disabled with an issue...

Reply:
ok, well we should remove the if statement above and replace it with your ones then, seems kinda dum to have both?